package.json

```diff
@@ -83,7 +83,7 @@
   },
```

Comment:
Version 0.13.2 is not updated in package.json. You should do it too (npm uses it for the release)